changelog/unreleased/features/6886.md

```diff
@@ -0,0 +1,3 @@
+- Added custom header x-mapbox-app-info to requests.
+- Fix upcoming route alerts incorrect position.
```

Comment:
```suggestion
- Fixed an issue with incorrect position of  `UpcomingRoadObject`s.
```
Is that true for all objects or only the ones that are not on the first leg? If only multi-leg routes were impacted, let's specify.